### PR TITLE
Fix for invalid redis auth

### PR DESCRIPTION
**Pull request by sdreyer**

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -118,13 +120,20 @@ jobs:
...
         # supported for use outside of 'steps'
         ports:
           - 6379:6379
 
+      redis-secure:
+        image: redis
+          # Hard coded port because environment variables not currently
+          # supported for use outside of 'steps'
+        ports:
+          - 6378:6379
```
Comment (Sami1309) [inline .github/workflows/testing.yml:135]: is this mapping correct?

**Review by Sami1309 — COMMENTED** [review]
besides comment looks good to me, merge if tests pass

File: .github/workflows/testing.yml (hunk 1 above)
Comment (sdreyer) [reply .github/workflows/testing.yml:135]: Yeah. For testing Ive setup a secure and insecure redis instance and test them both. Have to make one port different so they dont interfere

**Review by ahmadnazeri — DISMISSED** [review]
LGTM

**Pull request merged**